README.md

```diff
@@ -199,7 +201,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Request review based on files changes and/or groups the author belongs to
-        uses: necojackarc/auto-request-review@v0.12.0
+        uses: necojackarc/auto-request-review@v0.11.0
```

Comment:
Can you change it back to `v0.12.0`? I can update it when I release a new version.